### ISSUE TITLE
Panic raised by bad yacc file

### ISSUE THREAD

**Issue by snim2** [softdevteam/lrtable#63]
Here is the panic:
```sh
$ RUST_BACKTRACE=1 ./target/debug/lrpar text.l text.y lorem_ipsum.text 
thread 'main' panicked at 'Internal error', /home/snim2/.cargo/git/checkouts/lrtable-c613c5fde5035210/22ad6c6/src/lib/statetable.rs:90
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at /checkout/src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:60
             at /checkout/src/libstd/panicking.rs:355
   3: std::panicking::default_hook
             at /checkout/src/libstd/panicking.rs:371
   4: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:549
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:511
   6: lrtable::statetable::StateTable::new
             at /home/snim2/.cargo/git/checkouts/lrtable-c613c5fde5035210/22ad6c6/src/lib/statetable.rs:90
   7: lrtable::yacc_to_statetable
             at /home/snim2/.cargo/git/checkouts/lrtable-c613c5fde5035210/22ad6c6/src/lib/mod.rs:99
   8: lrpar::main
             at src/main.rs:105
   9: __rust_maybe_catch_panic
             at /checkout/src/libpanic_unwind/lib.rs:98
  10: std::rt::lang_start
             at /checkout/src/libstd/panicking.rs:433
             at /checkout/src/libstd/panic.rs:361
             at /checkout/src/libstd/rt.rs:56
  11: main
  12: __libc_start_main
  13: _start
```

And the lex/yacc inputs:

```sh
$ cat grammars/text.l 
%%
([^\n\r\t ])* WORD
[\n|\r\n][\n|\r\n]+ END_PARAGRAPH

$ cat grammars/text.y
%start document
%%
document : paragraph "END_PARAGRAPH" | document;
paragraph : sentence | paragraph;
sentence : "WORD" | sentence;
```

**Comment by ltratt** [comment]
This grammar is ambiguous, and not LR compatible, leading to an accept/reduce conflict. I need to think if we can easily catch the problem earlier in the transformation chain than the actual conflict.

**Comment by snim2** [comment]
I think this was fixed in #64 and so can be closed.
